src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,56 @@
+package command;
+/**
```

Comment:
I do not see any JavaDoc explanations for these methods. Perhaps there was a problem when committing or pushing?